operator/utils/utils.go

```diff
@@ -9,6 +9,14 @@ import (
 	v1 "k8s.io/api/core/v1"
 )
 
+func CreateGatewayUrlPrefix(namespace string, sdepName string) string {
```

Comment:
Nice one on abstracting these! :+1:

Reply:
Yeah, I think Clive added some changes to align with Ambassador URLs so when this is landed I'll add another pR to use this function across Ambassador and Istio